lib/encoder_actions.lua

```diff
@@ -1186,15 +1186,23 @@ function ea.move_rec_end(d)
   else
     res = d/rec_loop_enc_resolution
   end
-  if d <= 0 and util.round(rec[rec.focus].start_point,0.01) < util.round(rec[rec.focus].end_point + ((res)/lbr[params:get("live_buff_rate")]),0.01) then
-    rec[rec.focus].end_point = util.clamp(rec[rec.focus].end_point+((res)/lbr[params:get("live_buff_rate")]),(1+(8*(rec.focus-1))),(9+(8*(rec.focus-1))))
+  local min_end_point = 1+(8*(rec.focus-1))
+  local max_end_point = 9+(8*(rec.focus-1))
+  local resolved_res = res / lbr[params:get("live_buff_rate")]
+  local new_end_point = rec[rec.focus].end_point + resolved_res
```

Comment:
i did some mild cleanup here as well, extracting these re-used statements like `1+(8*(rec.focus-1))` into proper variables for clearer & easier re-use through the next few lines.

Reply:
raaaad, tytyty!!